### PR TITLE
Support shells in fetch

### PR DESCRIPTION
**Pull request by eepykate**

### CODE REVIEW

**Comment by eepykate** [comment]
Oh, and can we plz just remove the linter already :pleading_face:

File: db/sysinfo.go [hunk 1]
```diff
@@ -92,6 +93,7 @@ func TopSysinfoFields() ([]TopSysinfo, error) {
 		SELECT 1 as order, * FROM top_field('Distro')
 		UNION SELECT 2 as order, * FROM top_field('DeWm')
 		UNION SELECT 3 as order, * FROM top_field('DisplayProtocol')
+		UNION SELECT 3 as order, * FROM top_field('Shell')
```
Comment (Tanish2002) [inline db/sysinfo.go]: make it SELECT 4 and change all the below for that

**Review by Tanish2002 — CHANGES_REQUESTED** [review]

File: db/sysinfo.go (hunk 1 above)
Comment (eepykate) [reply db/sysinfo.go]: Good catch, I didn't even notice.

I did literally just copy-paste without thinking.

**Comment by eepykate** [comment]
Thank you!

**Pull request merged**